elements/lisk-utils/src/math/q.ts

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable no-bitwise */
+/*
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { MAX_FRAC, ONE, Q_OPERATION, TWO } from './constants';
+
+const numberToQ = (base: bigint, val: number): bigint => {
```

Comment:
This function handles conversion of a number to Q notation. Shouldn't it belong inside `Q` class, instead of keeping it as an external function?

Reply:
this is util function which doesn't use any context.
Also, it's used in the static function, so it cannot be method, so i just preferred to be an util function